ui/src/components/build/MobileClientBuildOverviewList.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { ListView } from 'patternfly-react';
+import MobileClientBuildListItem from './MobileClientBuildListItem';
+import NoBuildConfig from './NoBuildConfig';
+
+export const MobileClientBuildOverviewList = ({ buildConfigs, appName, config }) => (
+  <div>
+    {// TODO: Mobile builds are not avaialble for Xamarin.
+    buildConfigs.length === 0 ? (
+      <NoBuildConfig config={config} />
```

Comment:
IMO we don't need to pass `config` here. `CreateBuildConfigDialog` component can determine `clientType` and `clientId` by accessing redux store directly.

Reply:
passing `clientInfo` now, in redux store it's not available specifically what client app is selected, it's determined in `Client.js`